service/src/main/resources/db/migration/V218__add_occupancy_coefficient_under_3y.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE service_need_option ADD COLUMN occupancy_coefficient_under_3y numeric(4,2) DEFAULT 1.75;
```

Comment:
Why is there a `DEFAULT` value? It probably does more harm than good, because now you can insert a row without thinking about it.